src/Components/CryptoItem.tsx

```diff
@@ -0,0 +1,164 @@
+import React from 'react';
+import {View, Text, Image, StyleSheet, TouchableOpacity} from 'react-native';
+import AddButton from './AddButton';
+
+type props = {
+  setShowCrypto: (value: boolean) => void;
+};
+
+interface cryptoType {
+  id: string;
+  name: string;
+  symbol: string;
+  img: string;
+  price: number;
+  price_last_24_hours: number;
+}
+
+export const CryptoItem = ({setShowCrypto}: props) => {
+  const MarketCrypto: cryptoType = {
```

Comment:
rename to marketCrypto, in camelCase, and rename cryptoType to CryptoType in PascalCase, same with all the variables and Types/Interfaces